builder/index.html

```diff
@@ -156,7 +156,7 @@ <h1>You are not authenticated in ncf-builder</h1>
           <div class="col-xl-4 col-5 col-techniques" ng-click="toggleDisplay(true)">
             <div class="row flex-column">
               <div class="col-auto">
-                <div class="row" ng-if="ui.showTechniques">
```

Comment:
This was used to prevent the user from clicking on the "Import" button when the left menu is folded. We shouldn't remove it, or find another workaround for this bug

Reply:
I fixed by adding a css pointer-events, that disable click events https://css-tricks.com/almanac/properties/p/pointer-events/